onionbalance/hs_v3/params.py

```diff
@@ -48,3 +48,7 @@
 # Misc parameters
 
 DEFAULT_LOG_LEVEL = os.environ.get('ONIONBALANCE_LOG_LEVEL', 'warning')
+
+
+MAX_INTRO_POINTS=10
```

Comment:
Hmm, where did this 10 value come from? Can you consistently fit 10 intro points in a descriptor? If yes, let's change the name to `MAX_INTRO_POINTS_PER_DESCRIPTOR` and document it appropriately.